scripts/pre_commit_linter.py

```diff
@@ -717,9 +726,12 @@ def _check_comments(all_files):
 
                 if line.startswith('#') and not next_line.startswith('#'):
                     # Check that the comment ends with the proper punctuation.
-                    if (line[-1] not in
-                            ALLOWED_TERMINATING_PUNCTUATIONS) and (
-                                not any(word in line for word in EXCLUDED_PHRASES)):
+                    punctuation_is_allowed = line[-1] in (
```

Comment:
Hm, isn't this more like last_char_is_valid (or, better, last_char_is_invalid and then you don't need the not in the if clause)?

Reply:
I agree with this completely. Going with `last_char_is_invalid` then!

Reply:
Done.